### PR TITLE
add putzplan columns

### PR DESCRIPTION
**Pull request by Derstefan**

### CODE REVIEW

File: botMessages.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // bots messages
-const MESSAGES: { [key: string]: string } = {}
+const MESSAGES: { [key: string]: string } = {};
```
Comment (garciamarin) [inline botMessages.ts:2]: stop please

File: botMessages.ts [hunk 2]
```diff
@@ -15,13 +15,14 @@ I wait in an enforced slumber biding my time,
 until the fated moment when I shall be set free once again, 
 to reignite with my reminders upon your unorganized world! 🧞`;
 
-MESSAGES.SUBSCRIBED_FAIL = `Chat already susbscribed to /remiders`
+MESSAGES.SUBSCRIBED_FAIL = `Chat already susbscribed to /remiders`;
```
Comment (garciamarin) [inline botMessages.ts:18]: stop it

File: botMessages.ts [hunk 3]
```diff
@@ -15,13 +15,14 @@ I wait in an enforced slumber biding my time,
...
 
-MESSAGES.UNSUBSCRIBED_FAIL = `Your chat is not even on my list, bby 🧞`
+MESSAGES.UNSUBSCRIBED_FAIL = `Your chat is not even on my list, bby 🧞`;
```
Comment (garciamarin) [inline botMessages.ts:21]: just stap it please

File: index.ts [hunk 4]
```diff
@@ -1,96 +1,103 @@
 import TelegramBot, { Message } from "node-telegram-bot-api";
-import { config } from "dotenv"
+import { config } from "dotenv";
```
Comment (garciamarin) [inline index.ts:2]: stop here too plaese

File: utils.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
+function getWeeksCount(date: Date): number {
+  const epoch = new Date("2024-01-01T00:00:00Z");
+
+  // Calculate the time difference in milliseconds between the given date and the epoch
```
Comment (garciamarin) [inline utils.ts]: lets get rid of this comment!

File: utils.ts [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  const timeDifference = date.getTime() - epoch.getTime();
+
+  // Convert the time difference from milliseconds to weeks
```
Comment (garciamarin) [inline utils.ts]: stop it

File: utils.ts [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+  const weeksDifference = timeDifference / (1000 * 60 * 60 * 24 * 7);
+
+  // Return the calculated number of weeks
```
Comment (garciamarin) [inline utils.ts]: stop it chat

File: index.ts [hunk 8]
```diff
@@ -1,96 +1,103 @@
...
-            bot.sendMessage(chatId, MESSAGES.REMINDERS_CLEANING_PLAN)
-        } else if (currentDate.getDay() === 5) { // Friday
-            bot.sendMessage(chatId, MESSAGES.REMINDERS_VEGTABLE_BOX)
-        }
+  const currentDate = new Date();
+  const reminders = (chatId: number) => {
+    if (currentDate.getDay() === 1) {
+      // Monday with (0 - Sunday, 1 - Monday, ...)
+      if (isEvenWeek(currentDate)) {
+        bot.sendMessage(chatId, MESSAGES.REMINDERS_CLEANING_PLAN_AWEEK);
```
Comment (garciamarin) [inline index.ts]: I rather have more descriptive names here like: 

`REMINDERS_CLEANING_PLAN_EVEN_WEEK`

File: botMessages.ts [hunk 9]
```diff
@@ -15,13 +15,14 @@ I wait in an enforced slumber biding my time,
...
 
-MESSAGES.REMINDERS_CLEANING_PLAN = `🧹🪥 This is an automated Putzplan reminder! 🧼🫧`
+MESSAGES.REMINDERS_CLEANING_PLAN_EVEN_WEEK = `🧹🪥 This is an automated Putzplan reminder! Left nothing behind! 🧼🫧`;
+MESSAGES.REMINDERS_CLEANING_PLAN = `🧹🪥 This is an automated Putzplan reminder! Shine right like a diamond 🧼🫧`;
```
Comment (garciamarin) [inline botMessages.ts]: can we make the other week `ODD_WEEk`

Comment (garciamarin) [reply botMessages.ts]: maybe we can simplify it a bit here and in the `index.ts` with the logic and dupplication of strings. What do you think? :)

```js
`MESSAGES.REMINDERS_CLEANING_PLAN` = `🧹🪥 This is an automated Putzplan reminder! 🧼🫧\n`
`MESSAGES.EVEN_WEEK` = 'Left nothing behind!  ↩️ '
`MESSAGER.ODD_WEEK` ='Shine right like a diamond ➡️' 
```
Like this it also has the advantage that if we want to modify the basic message, we dont have to maintain the stings in two places, and we only have to change one. 

Then we can use these stings in `index.ts` like 

 ```js
 const currentDate = new Date()
  const reminders = (chatId: number) => {
    if (currentDate.getDay() === 1) {
      const message = 
      MESSAGES.REMINDERS_CLEANING_PLAN_AWEEK + 
        isEvenWeek(currentDate)  ?
          MESSAGES.EVEN_WEEK : 
          MESSAGES.ODD_WEEK 
      // Monday with (0 - Sunday, 1 - Monday, ...)
      if () {
        bot.sendMessage(chatId, message)
```
 
...its just a suggestion though

**Review by garciamarin — CHANGES_REQUESTED** [review]

**Comment by garciamarin** [comment]
@Derstefan I think that the `getWeeksCount` function will return for a given day, the number of the week this day is counting from the 1st of january of 2024 on, which might not be necessarily the same that the calendar week which that day is on, right?   They should only coincide if the year started on 2024. 
Moreover, this will just work for the 2024. Maybe we have to change it a little bit?

**Comment by garciamarin** [comment]
very cool!

File: botMessages.ts (hunk 9 above)
Comment (Derstefan) [reply botMessages.ts]: hm i wouldn't generalize it that much, its easyer to read and to change it, if we let it separate like this.
But if you really want it I change it darling  :*

**Review by garciamarin — APPROVED** [review]
🚀

**Pull request merged**